lib/runner.js

```diff
@@ -52,11 +56,29 @@ class Runner extends Emittery {
 			if (!scheduledStart) {
 				scheduledStart = true;
 				process.nextTick(() => {
+					this.callsiteLineNumbers.push(Infinity);
+					this.callsiteLineNumbers.sort((a, b) => a - b);
```

Comment:
Let's do this work inside the `start()` method instead.

We should deduplicate the line numbers as well. It's an edge case but you could specify two tests on the same line.